editor/doc/editor_help.cpp

```diff
@@ -53,17 +53,17 @@
 #include "editor/themes/editor_scale.h"
 #include "scene/gui/line_edit.h"
 
-#include "modules/modules_enabled.gen.h" // For gdscript, mono.
+#include "modules/modules_enabled.gen.h" // For gdscript, mono. // validator: ignore // FIXME: Legacy include.
 
 // For syntax highlighting.
 #ifdef MODULE_GDSCRIPT_ENABLED
-#include "modules/gdscript/editor/gdscript_highlighter.h"
-#include "modules/gdscript/gdscript.h"
+#include "modules/gdscript/editor/gdscript_highlighter.h" // validator: ignore // FIXME: Legacy include.
+#include "modules/gdscript/gdscript.h" // validator: ignore // FIXME: Legacy include.
```

Comment:
These would have to have have the GDScript specific things be moved to the module with some form of method to access it indirectly instead, perhaps via `script_language`